salesapp.js

```diff
@@ -96,6 +98,35 @@ function runAllStoreData() {
   }
 };
 
+function runStoreHourTotals(){
+  var total = 0;
+
+  for (var i = 0; i < storeTimes.length - 2; i++) {
+    var hourlyTotal = 0;
+    for (var j = 0; j < storeArray.length; j++) {
+      console.log(storeArray[j].hourlyCookies[i]);
+      total += storeArray[j].hourlyCookies[i];
+      hourlyTotal += storeArray[j].hourlyCookies[i];
+
+    }
+    hourlyTotalsStorage.push(hourlyTotal);
+  }
+  console.log('TOTAL:', total);
+  console.log('HourlyTotal', hourlyTotalsStorage);
+}
+
+var sumEverything = function(){
```

Comment:
What's the difference between this and the `total` variable in `runStoreHourTotals`?